app/services/policies/policies_list_service.rb

```diff
@@ -4,16 +4,21 @@
 require 'set'
 
 class Policies::PoliciesListService
+
+  class PoliciesListServiceError < RuntimeError; end
+
   def self.call(account, builtin: true)
     list = PolicyList.new
-    if builtin
-      response = ::JSONClient.get(ThreeScale.config.sandbox_proxy.apicast_registry_url)
-      return unless response.ok?
-
-      list.merge!(PolicyList.from_hash(response.body['policies']))
-    end
-    list.merge!(PolicyList.new(account.policies)) if account.provider_can_use?(:policy_registry)
+    list.merge! fetch_policies_from_apicast if builtin
+    list.merge! account.policies if provider_can_use_policies
     list.to_h
+  rescue PoliciesListServiceError => error
+    Rails.logger.error { error }
```

Comment:
```suggestion
    Rails.logger.error { error } and return
```
just one line is enough for this 😄